spec/generators/paper_trail/install_generator_spec.rb

```diff
@@ -102,4 +102,27 @@
       )
     end
   end
+
+  describe "`--uuid` option set to `true`" do
+    before do
+      prepare_destination
+      run_generator %w[--uuid]
+    end
+
+    expected_item_id_type = "string"
```

Comment:
Thanks for writing a test, but it's unusual IME to define a local variable in a `describe` block. Please move this variable into the `it` block.